Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,11 +34,71 @@ public static class TextTable
         /// <summary>
         /// Build a table for given message with given padding.
         /// Padding means how many spaces will a message be wrapped with.
-        /// Table itself is made of: "+-" symbols. 
+        /// Table itself is made of: "+-" symbols.
         /// </summary>
         public static string Build(string message, int padding)
         {
-            return "";
+            if (string.IsNullOrEmpty(message))
+            {
+                return "";
+            }
+
+            var longestWordInMessageSize = LongestWordInMessageSize(message);
+
+            var newHorizontalBorder = new StringBuilder();
+            newHorizontalBorder.Append(cornerChar).Append(horizontalBorderChar, longestWordInMessageSize + (padding * 2)).Append(cornerChar).Append(Environment.NewLine);
+
+            var newEmptyLine = new StringBuilder();
+            newEmptyLine.Append(verticalBorderChar).Append(' ', longestWordInMessageSize + (padding * 2)).Append(verticalBorderChar).Append(Environment.NewLine);
+
+            var sb = new StringBuilder();
+
+            sb.Append(newHorizontalBorder);
+
+            for (var i = 0; i < padding; i++)
+            {
+                sb.Append(newEmptyLine);
+            }
+
+            var wordsInMessage = message.Split(dividerChar);
+
+            foreach (var word in wordsInMessage)
+            {
+                sb.Append(verticalBorderChar);
+                sb.Append(' ', padding);
+                sb.Append(word);
+                sb.Append(' ', longestWordInMessageSize - word.Length);
+                sb.Append(' ', padding);
+                sb.Append(verticalBorderChar);
+                sb.Append(Environment.NewLine);
+            }
+
+            for (var i = 0; i < padding; i++)
+            {
+                sb.Append(newEmptyLine);
+            }
+
+            sb.Append(newHorizontalBorder);
+
+            return sb.ToString();
+        }
+
+        private const string dividerChar = "\r\n";
+        private const char cornerChar = '+';
+        private const char horizontalBorderChar = '-';
+        private const char verticalBorderChar = '|';
```

Comment:
If I am not mistaken the convention wants this to be the first thing in your class and from everything you will learn until the end of the course this would probably one of the first items overall. Kai may want to confirm this.

My reason for picking this out is that you use those constants in the function above which makes it harder to follow your thoughts. My understanding of order goes as this:
public
internal
private

constants
variables
functions

Reply:
Yes, I know the convention for this. I just positioned those at the end because the wall of comments at the beginning.